Yosemite/Yosemite/Model/Mocks/ActionHandlers/MockAppSettingsActionHandler.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import Storage
+
+struct MockAppSettingsActionHandler: MockActionHandler {
+    typealias ActionType = AppSettingsAction
+
+    let objectGraph: MockObjectGraph
+    let storageManager: StorageManagerType
+
+    func handle(action: ActionType) {
+        switch action {
+            case .loadInitialStatsVersionToShow(let siteId, let onCompletion):
+                loadInitialStatsVersionToShow(siteId: siteId, onCompletion: onCompletion)
+            case .setStatsVersionLastShown(let siteId, let statsVersion):
+                setStatsVersionLastShown(siteId: siteId, statsVersion: statsVersion)
+            case .loadFeedbackVisibility(let type, let onCompletion):
+                loadFeedbackVisibility(type: type, onCompletion: onCompletion)
+            case .setInstallationDateIfNecessary(let date, let onCompletion):
+                setInstallationDateIfNecessary(date: date, onCompletion: onCompletion)
+
+            default: unimplementedAction(action: action)
+        }
+    }
+
+    func loadInitialStatsVersionToShow(siteId: Int64, onCompletion: (StatsVersion?) -> Void) {
+        onCompletion(.v4)
+    }
+
+    func setStatsVersionLastShown(siteId: Int64, statsVersion: StatsVersion) {
+        // TODO – we should persist this somewhere
```

Comment:
nit: 💡 wdyt about using a warning to not forget about this later?
```suggestion
        #warning("TODO – we should persist this somewhere")
```

(And same for other `// TODO – xxx` comments I've seen in other mocks ofc)

Reply:
I've seen this pattern before, but we don't allow new warnings in PRs, so I can't use it 😅

If we were looking to fully build out the backend we'd persist it to a `plist` but it's not _required_. However, leaving the message and empty function body there hints that it's a future improvement we could make without requiring it or making a lot of noise 🤷‍♂️

Reply:
Would then maybe `// TODO: xxx` (with the specific syntax that Xcode recognizes, i.e. exact `// TODO:` prefix) be better? Given that this is (1) recognized by Xcode IDE (2) can be turned into a warning on demand [by a SwiftLint rule](https://realm.github.io/SwiftLint/todo.html) (there's even [another one that allows to put an expiration date on a TODO](https://realm.github.io/SwiftLint/expiring_todo.html) so the warning is only triggered after said date…)

Reply:
Addressed in 42f65e17c9df539736946c65e3c0a4f2155fe56f since yeah, there's no state. Just replaced with a no-op until it's removed!